pkg/operator/controllers/guardrails/policies/gktemplates-src/aro-deny-upgradeconfig/src.rego

```diff
@@ -0,0 +1,25 @@
+package arodenyupgradeconfig
+
+import data.lib.common.is_exempted_account
+import future.keywords.in
+
+
+# Use object
+# To retrieve from a different resource, data.inventory.namespace["openshift-managed-upgrade-operator"]["v1"]["ConfigMap"]["managed-upgrade-operator-config"]["data"]["config.yaml"]
+violation[{"msg": msg}] {
+	input.review.operation in ["CREATE", "UPDATE", "DELETE"]
+
+	# ## Check user type
+	not is_exempted_account(input.review)
+
+	# ## If regular user and
+	# ## has NO cloud.openshift.com entry in openshift-config/pull-secret Secret
+	# ## ALLOW EDITING
+
+	# ## If regular user and 
+	# ## HAS cloud.openshift.com entry (`source: OCM` indicates pull-secret exists) in openshift-config/pull-secret Secret
+	# ## NOT ALLOWED
+	config_data := data.inventory.namespace["openshift-managed-upgrade-operator"]["v1"]["ConfigMap"]["managed-upgrade-operator-config"]["data"]["config.yaml"]
```

Comment:
so my understanding here is, we are not able to retrieve the `config.yaml` data directly from `input review` object right?